build_tools/android/common/noisy_benchmarks.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""A list of noisy benchmarks and their average thresholds."""
+
+import re
+
+NOISY_BENCHMARKS = [
+    # (regular expression to match the benchmark name, average threshold)
```

Comment:
Please expand on what "average threshold" is. It looks like a string that just gets passed through to the dashboard as json? When would some set this to "100%" vs "50%"?

It seems very strange to pass a number as a string, rather than as a number then add the `%` when actually using it for display.

Reply:
I explained more in the comment to make it clear. Specifically for your question:

> When would some set this to "100%" vs "50%"?

It depends on how noisy the particular benchmark is. If it's fluctuating with in 50% range then we can use that. For the current GPU noisy benchmarks, they are all 80%+ - 100% so I use 100% here.

> It seems very strange to pass a number as a string, rather than as a number then add the % when actually using it for display.

Actually Dana supports both percentage or absolute value for specifying the range. Can put 100 here but it will mean 100 in absolute value. String here is actually better. I explained more in the comment.

Reply:
Okay, my point about number vs string is that if another script were to inspect the value and make some decision based on it, it would want a number. The number portion could be parsed from a string, but it's easier to keep data in its base form then only format it where it gets displayed.